8.x/core-concepts/annotations.md

```diff
@@ -85,62 +85,62 @@ public function test()
 
 First, install the package:
 
-```bash{1}
+```bash
 composer require larammerce/annotation-parser
 ```
 
```

Comment:
As these classes are constructed based on `reflection in PHP`, You can mention it here, and refer to the main website for more information :
https://www.php.net/manual/en/book.reflection.php